pkg/types/powervs/defaults/platform.go

```diff
@@ -1,16 +1,27 @@
 package defaults
 
 import (
+	"crypto/rand"
+	"fmt"
+	"math/big"
+
 	"github.com/openshift/installer/pkg/ipnet"
 	"github.com/openshift/installer/pkg/types/powervs"
 )
 
 var (
 	// DefaultMachineCIDR is the PowerVS default IP address space from
 	// which to assign machine IPs.
-	DefaultMachineCIDR = ipnet.MustParseCIDR("192.168.0.0/24")
+	DefaultMachineCIDR *ipnet.IPNet
```

Comment:
What about we still set the `192.168.0.0/24` value here in case this value is used before `SetPlatformDefaults`? Worst case scenario we get a subnet conflict instead of dereferencing a `nil` value.